tools/dexnode/src/hosts/chrome.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import path from "path";
+import { type Host } from "../host.js";
+import { HostFlags } from "../hostFlags.js";
+
+export const HOST_CHROME_STABLE: Host<"chrome"> = {
+    flagName: "chrome",
+    flags:
+        HostFlags.ChromiumArguments |
+        HostFlags.UseLatestV8Version,
+    paths: {
+        win32: [
+            "C:/Program Files/Google/Chrome/Application/chrome.exe",
+            "C:/Program Files (x86)/Google/Chrome/Application/chrome.exe",
+            "chrome"
+        ],
+        darwin: [
+            "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome",
+        ],
+        linux: [
+            "/usr/bin/google-chrome-stable",

```

Comment:
One thing I might suggest is reversing the order; at least on my machine, chrome/edge are on PATH as wrapper scripts that allow me to pass in some default flags for my system, and so trying to execute them directly from `/usr/bin` or `/opt` may or may not do the right thing.

Reply:
> [...] wrapper scripts that allow me to pass in some default flags for my system [...]

That sounds like it could potentially cause problems. How would I know whether those flags might interfere with log generation?

Reply:
The flags are for things like "enable dark mode" and "enable wayland support", or disabling certain features like the bing bar or whatever other things people don't want, so I doubt that anyone will end up in a situation where they're setting V8 flags.